apis/apps/v1alpha1/apimanager_types.go

```diff
@@ -1438,6 +1442,28 @@ func (a *APIManager) GetApicastCustomPoliciesSecretRefs() []*v1.LocalObjectRefer
 	return secretRefs
 }
 
+func (a *APIManager) GetSystemDatabaseSecretRefs() []*v1.LocalObjectReference {
+	secretRefs := []*v1.LocalObjectReference{}
+	if a.IsSystemDatabaseTLSEnabled() {
```

Comment:
@austincunningham , I'd like to ask - maybe system-database and zync  secrets need to be watched always (with TLS and without) ? I thought about this while starting add system-redis and backend-redis ... Maybe ALL 3scale secrets need to be added finally (but not in TLS PRs)?  I think I will add redis secrets not conditionally meanwhile. Thank you

Reply:
It's a far question as it no longer a matter of the user adding the annotation to get the watchby logic to work. I think you are right we should have it on by default.

Reply:
discussed with the team , the watch-by annotations should be in place but the watch-by label would have to be added by the end user.

Reply:
thank you